aio.core/aio/core/functional/utils.py

```diff
@@ -9,10 +9,15 @@
 
 from trycast import trycast  # type:ignore
 
-import orjson as json
-
 from aio.core.functional import exceptions
 
+# condition needed due to https://github.com/bazelbuild/rules_python/issues/622
+import sys
+if 'darwin' not in sys.platform:
+    import orjson as json
+else:
+    import json
```

Comment:
i have not tested if this else part works ;)

Reply:
i would just try/except on ModuleError and/or maybe ImportError

Reply:
just remembering that this is a bazel issue rather than mac so we defo want try/except - this should work in other mac envs (and may not work in some non mac envs)